packages/trpc/server/routers/viewer/teams/listMembers.handler.ts

```diff
@@ -36,6 +36,8 @@ export const listMembersHandler = async ({ ctx, input }: ListMembersOptions) =>
               id: true,
               name: true,
               username: true,
+              email: true,
```

Comment:
When submitting the routing form we query the DB again for the user email. Let's not include the user email in the list member's query then.

Reply:
Done